src/pages/community.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+
+import styles from './index.module.scss';
+import clsx from 'clsx';
+
+function Community() {
+  return (
+    <Layout title="Community">
+      <div className={clsx('subtron', styles.subtron)}>
+          <h1>Electron Community</h1>
+          <p>Resources for connecting with people working on Electron.</p>
+        </div>
+      <div className={clsx('community_section', styles.community_section)}>
```

Comment:
Since we're using a CSS module, you don't need to actually add `community_section` as a class here